py3status/modules/whatismyip.py

```diff
@@ -20,6 +20,7 @@
 
 Format placeholders:
     {country} display the country
+    {countryCode} display the country code
```

Comment:
how about `{country_code}`?  This is how we seem to do this rather than camelCase